great_expectations/core/expectation_suite.py

```diff
@@ -518,42 +518,21 @@ def patch_expectation(
         self.expectations[found_expectation_indexes[0]].patch(op, path, value)
         return self.expectations[found_expectation_indexes[0]]
 
-    def add_expectation(
+    def _add_expectation(
         self,
         expectation_configuration: ExpectationConfiguration,
+        send_usage_event: bool = False,
```

Comment:
I'm leaning to not having any default value for this flag, so that it is always obvious when the caller is suppressing usage events